+++
title = "Structure, variation, and assembly of the root-associated microbiomes of rice"
date = "2015-02-24"

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
authors = ["**Joseph Edwards**", "Cameron Johnson", "Christian Santos-Medell√≠n", "Eugene Lurie", "Natraj Kumar Podishetty", "Srijak Bhatnagar", "Jonathan A. Eisen", "Venkatesan Sundaresan"]

# Publication type.
# Legend:
# 0 = Uncategorized
# 1 = Conference proceedings
# 2 = Journal
# 3 = Work in progress
# 4 = Technical report
# 5 = Book
# 6 = Book chapter
publication_types = ["2"]

# Publication name and optional abbreviated version.
publication = "*Proceedings of the National Academies of Sciences*"
publication_short = "In *PNAS*"

# Abstract and optional shortened version.
abstract = "Plants depend upon beneficial interactions between roots and microbes for nutrient availability, growth promotion, and disease suppression. High-throughput sequencing approaches have provided recent insights into root microbiomes, but our current understanding is still limited relative to animal microbiomes. Here we present a detailed characterization of the root-associated microbiomes of the crop plant rice by deep sequencing, using plants grown under controlled conditions as well as field cultivation at multiple sites. The spatial resolution of the study distinguished three root-associated compartments, the endosphere (root interior), rhizoplane (root surface), and rhizosphere (soil close to the root surface), each of which was found to harbor a distinct microbiome. Under controlled greenhouse conditions, microbiome composition varied with soil source and genotype. In field conditions, geographical location and cultivation practice, namely organic vs. conventional, were factors contributing to microbiome variation. Rice cultivation is a major source of global methane emissions, and methanogenic archaea could be detected in all spatial compartments of field-grown rice. The depth and scale of this study were used to build coabundance networks that revealed potential microbial consortia, some of which were involved in methane cycling. Dynamic changes observed during microbiome acquisition, as well as steady-state compositions of spatial compartments, support a multistep model for root microbiome assembly from soil wherein the rhizoplane plays a selective gating role. Similarities in the distribution of phyla in the root microbiomes of rice and other plants suggest that conclusions derived from this study might be generally applicable to land plants."
abstract_short = "Land plants continuously contact beneficial, commensal, and pathogenic microbes in soil via their roots. There is limited knowledge as to how the totality of root-associated microbes (i.e., the microbiome) is shaped by various factors or its pattern of acquisition in the root. Using rice as a model, we show that there exist three different root niches hosting different microbial communities of eubacteria and methanogenic archaea. These microbial communities are affected by geographical location, soil source, host genotype, and cultivation practice. Dynamics of the colonization pattern for the root-associated microbiome across the three root niches provide evidence for rapid acquisition of root-associated microbiomes from soil, and support a multistep model wherein each root niche plays a selective role in microbiome assembly."

# Featured image thumbnail (optional)
image_preview = ""

# Is this a selected publication? (true/false)
selected = true

# Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter the filename (excluding '.md') of your project file in `content/project/`.
projects = ["example-external-project"]

# Links (optional).
url_pdf = "http://www.pnas.org/content/112/8/E911.full.pdf"
url_code = "https://github.com/RiceMicrobiome/Edwards-et-al.-2014"


# Custom links (optional).
#   Uncomment line below to enable. For multiple links, use the form 
#url_custom = [{name = "Custom Link", url = "http://example.org"}]

# Does the content use math formatting?
math = true

# Does the content use source code highlighting?
highlight = true

# Featured image
# Place your image in the `static/img/` folder and reference its filename below, e.g. `image = "example.jpg"`.
[header]
image = "headers/PNAS2015.png"
caption = "My caption :smile:"

+++